person.rb

```diff
@@ -28,4 +28,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Great job working on the functionality for this method but I think the rental is related with both the person and the book so I can see it would be better if you implement it accordingly so you can add something like this since the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 🚀💯🚀✅


```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```